tests/Faker.Tests/Common/NameTests.cs

```diff
@@ -27,14 +29,30 @@ public void Should_Get_Suffix()
 		}
 
 		[Test]
-		[Repeat(100000)]
+		[Repeat(1)]
 		public void Compute_Time_Elapsed_By_The_First_Method()
 		{
-			Name.First();
+			var sw = new Stopwatch();
+			sw.Start();
+			for (int i = 0; i < 100000; i++)
+			{
+				Name.First();
+			}
+			sw.Stop();
+			Console.WriteLine("Elapsed time without caching {0}msec", sw.ElapsedMilliseconds);
+			sw.Reset();
+
+			sw.Start();
+			for (int i = 0; i < 100000; i++)
+			{
+				Name.FirstCached();
+			}
+			sw.Stop();
+			Console.WriteLine("Elapsed time with caching {0}msec", sw.ElapsedMilliseconds);
```

Comment:
Although this would be great as a benchmark in a console application,
it doesn't fit as a unit test.
A unit test is supposed to test something with a possibility of failing if something is wrong,
This just outputs the how long it took (which would be great as a benchmark)